docs/table/mixin_columns.rst

```diff
@@ -141,6 +141,16 @@ You can conveniently convert |Table| to |QTable| and vice-versa::
    ordinary `~astropy.table.Table` then it gets converted to an ordinary
    `~astropy.table.Column` with the corresponding ``unit`` attribute.
 
+.. attention::
+
+  When a column of ``int`` ``dtype`` is converted to `~astropy.units.Quantity`,
```

Comment:
Not important, but for the sake of consistency... I won't defend this convention, but most sphinx and numpydoc sections in directives are indented 3 or 4 spaces, rarely 2.

Reply:
good catch - I didn't notice the difference.